browser_use/browser/session.py

```diff
@@ -390,10 +394,8 @@ async def setup_browser_via_browser_pid(self) -> None:
 		# we could automatically relaunch the browser process with that arg added here, but they may have tabs open they dont want to lose
 		self.cdp_url = self.cdp_url or f'http://localhost:{debug_port}/'
 		logger.info(f'🌎 Connecting to existing local browser process: browser_pid={self.browser_pid} on {self.cdp_url}')
-		self.browser = self.browser or await self.playwright.chromium.connect_over_cdp(
-			self.cdp_url,
-			**self.browser_profile.kwargs_for_connect().model_dump(),
-		)
+		assert self.driver is not None, 'Driver object is not set'
+		self.browser = self.browser or await self.setup_browser_via_cdp_url()
```

Comment:
don't re-use that function, there's a reason it was re-implemented here, the error handling cases are different and we want to add some different connection args here soon for local pid-based connections